learning_journal/templates/edit.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends "templates/layout.jinja2" %}
+{% block body %}
+<form action="{{ request.route_url('action', action='edit') }}" method="POST">
+{% for field in form %}
+  {% if field.errors %}
+    <ul>
+    {% for error in field.errors %}
+        <li>{{ error }}</li>
+    {% endfor %}
+    </ul>
+  {% endif %}
+  {% if field.widget.input_type != 'hidden' %}
+    <p>{{ field.label(class_="form-label") }}: {{ field(class_="form-field-text") }}</p>
+  {% endif %}
```

Comment:
doesn't this prevent the input that holds the 'id' from rendering into your template at all?  that seems problematic to me.  Especially since the action URL you're rendering above does not contain the object ID at all.  How are you getting the ID back to the edit_view when this form is submitted?